examples/tenant.yaml

```diff
@@ -113,6 +113,12 @@ spec:
       #     cpu: 500m
       #      memory: 16Gi
 
+      ## Configure security context 
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        runAsNonRoot: true
```

Comment:
Why support root as true? We should never need to run console or minio as root.

Reply:
This is not a Operator / Tenant feature. I am just giving an example to disable running as root user in the `securityContext`.